src/contexts/Api/types.ts

```diff
@@ -24,12 +24,19 @@ export interface APIConstants {
   existentialDeposit: BigNumber;
   fastUnstakeDeposit: BigNumber;
   poolsPalletId: U8aLike;
+  ss58Prefix: BigNumber;
+}
+
+export interface APIChainState {
+  chain: string;
+  version: string;
```

Comment:
We could also make these optional but I don't that that would be correct; we always need to know if a chain is assigned or not.

```suggestion
  chain: string | undefined;
  version: string | undefined;
```